crates/bevy_asset/src/loader.rs

```diff
@@ -503,57 +769,6 @@ impl<'a> LoadContext<'a> {
         Ok(bytes)
     }
 
-    /// Retrieves a handle for the asset at the given path and adds that path as a dependency of the asset.
-    /// If the current context is a normal [`AssetServer::load`], an actual asset load will be kicked off immediately, which ensures the load happens
-    /// as soon as possible.
-    /// "Normal loads" kicked from within a normal Bevy App will generally configure the context to kick off loads immediately.
-    /// If the current context is configured to not load dependencies automatically (ex: [`AssetProcessor`](crate::processor::AssetProcessor)),
-    /// a load will not be kicked off automatically. It is then the calling context's responsibility to begin a load if necessary.
-    pub fn load<'b, A: Asset>(&mut self, path: impl Into<AssetPath<'b>>) -> Handle<A> {
```

Comment:
I would suggest leaving this method on `LoadContext`, since it will suffer the most ergonomically from the introduction of the builder pattern. Perhaps with some new comments indicating that if you need special loading beahviour, check `loader()`.